GameBoard/Models/FriendsSidebar/FriendsSidebarViewModel.cs

```diff
@@ -1,27 +1,43 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
-using GameBoard.Models.User;
+using GameBoard.Models.Groups;
 
 namespace GameBoard.Models.FriendsSidebar
 {
     public class FriendsSidebarViewModel
     {
-        public IEnumerable<UserViewModel> Friends { get; }
+        public IEnumerable<GroupViewModel> Groups { get; }
 
         public bool Toggled { get; }
 
+        public bool GroupInviteEnabled { get; }
+
+        public int GameEventId { get; }
+
+        public string GameEventName { get; }
+
+        public int GroupId { get; }
```

Comment:
Why do we need all this? Friends sidebar has no interest in game events. And what does `GroupId` even mean in this context?

Reply:
Resolved

Reply:
Also, removed GroupId